src/figuresClasses.ts

```diff
@@ -1,19 +1,70 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape = 'triangle';
 
-}
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0
+      || this.a + this.b <= this.c || this.a + this.c <= this.b
+      || this.b + this.c <= this.a) {
+      throw new Error('This figure is not a triangle');
```

Comment:
better to have it more descriptive, as it would be difficult to understand what exactly is wrong with my input in that case